rune/libenclave/internal/runtime/pal/api_linux_v2.go

```diff
@@ -0,0 +1,117 @@
+package enclave_runtime_pal // import "github.com/opencontainers/runc/libenclave/internal/runtime/pal"
+
+/*
+#include <stdlib.h>
+
+static int palCreateProcessV2(void *sym, const char *exe, const char *argv[],
+			    const char *envp[], int stdin, int stdout,
+			    int stderr, int *pid)
+{
+	typedef struct {
+                int stdin, stdout, stderr;
+        } pal_stdio_fds;
+
+	typedef struct {
+		const char *path;
+		const char **argv;
+		const char **env;
+		pal_stdio_fds *fds;
+		int *pid;
+	} pal_create_process_args;
```

Comment:
Use the anonymous struct to define fds and create_process_args, such as:
```C
struct {
 		const char *path;
 		const char **argv;
 		const char **env;
 		pal_stdio_fds *fds;
 		int *pid;
} create_process_args = {
 		exe,
 		argv,
 		envp,
 		&fds,
 		pid,
};
```

Reply:
pal_create_process_args is used to define the function pointer type, not anonymous.